examples/distributed/graphlearn_for_pytorch/dist_train_sage_sup_config.yml

```diff
@@ -0,0 +1,19 @@
+nodes:
+  - 0.0.0.0
+  - 1.1.1.1
+ports: [22, 22]
+python_bins:
+  - /path/to/python
+  - /path/to/python  # path to python with GLT's env
+usernames:
+  - your_username_for_node_0
+  - your_username_for_node_1
+dataset: ogbn-products
+in_channel: 100
+out_channel: 47
+dst_paths:
+  - /path/to/pytorch_geometric
+  - /path/to/pytorch_geometric
+visible_devices:
+  - 0,1,2,3
+  - 4,5,6,7
```

Comment:
Can we provide some more documentation in this file? Especially since it is a user-facing configuration

Reply:
Updated with more elaborate documentation